std/typecons.d

```diff
@@ -2638,33 +2638,36 @@ unittest
 
 /**
 $(D WhiteHole!Base) is a subclass of $(D Base) which automatically implements
-all abstract member functions as throw-always functions.  Each auto-implemented
-function fails with throwing an $(D Error) and does never return.  Useful for
-trapping use of not-yet-implemented functions.
+all abstract member functions as functions that always fail. These automatically
+implemented function simply throw an $(D Error) and never return. `Whitehole` is 
```

Comment:
"implemented function" -> "implemented functions". Currently you're using both singular and plural in the same sentence.